### ISSUE TITLE
Travis CI not reading exit code for yarn test

### ISSUE THREAD

**Issue by ningyuansg** [source-academy/cadet-frontend#275]
This one is pretty embarrassing. The travis CI command to run is `yarn test-coveralls` which is an alias for `./scripts/coverage-fix.sh && react-scripts-ts test --env=jsdom --coverage --coverageReporters=text-lcov | coveralls` which returns the exit code for `coveralls` which ignores test errors.

**Comment by ningyuansg** [comment]
E.g. tests have been passing in the local pre-push hook but not in travis.

```
FAIL src/components/assessment/__tests__/index.tsx (40.413s)
  â—Assessment page with multiple loaded missions renders correctly

    expect(value).toMatchSnapshot()

    Received value does not match stored snapshot 1.

    - Snapshot
    + Received

    @@ -89,11 +89,11 @@
                    time
                  </title>
                  <path d="M8 0C3.58 0 0 3.58 0 8s3.58 8 8 8 8-3.58 8-8-3.58-8-8-8zm0 14c-3.31 0-6-2.69-6-6s2.69-6 6-6 6 2.69 6 6-2.69 6-6 6zm1-6.41V4c0-.55-.45-1-1-1s-1 .45-1 1v4c0 .28.11.53.29.71l2 2a1.003 1.003 0 0 0 1.42-1.42L9 7.59z" fillRule="evenodd" />
                </svg>
              </Blueprint2.Icon>
    -         Due: 18/5, 13:24
    +         Due: 18/5, 5:24
            </div>
          </Text>
          <NavLink to="/academy/missions/0/0" activeClassName="active" ariaCurrent="true">
            <Route path="/academy/missions/0/0" exact={[undefined]} strict={[undefined]} location={[undefined]}>
              <Link to="/academy/missions/0/0" className={[undefined]} style={[undefined]} aria-current={false} replace={false}>
    @
                    time
                  </title>
                  <path d="M8 0C3.58 0 0 3.58 0 8s3.58 8 8 8 8-3.58 8-8-3.58-8-8-8zm0 14c-3.31 0-6-2.69-6-6s2.69-6 6-6 6 2.69 6 6-2.69 6-6 6zm1-6.41V4c0-.55-.45-1-1-1s-1 .45-1 1v4c0 .28.11.53.29.71l2 2a1.003 1.003 0 0 0 1.42-1.42L9 7.59z" fillRule="evenodd" />
                </svg>
              </Blueprint2.Icon>
    -         Due: 18/5, 13:24
    +         Due: 18/5, 5:24
            </div>
          </Text>
          <NavLink to="/academy/missions/1/0" activeClassName="active" ariaCurrent="true">
            <Route path="/academy/missions/1/0" exact={[undefined]} strict={[undefined]} location={[undefined]}>
              <Link to="/academy/missions/1/0" className={[undefined]} style={[undefined]} aria-current={false} replace={false}>
    @
                    time
                  </title>
                  <path d="M8 0C3.58 0 0 3.58 0 8s3.58 8 8 8 8-3.58 8-8-3.58-8-8-8zm0 14c-3.31 0-6-2.69-6-6s2.69-6 6-6 6 2.69 6 6-2.69 6-6 6zm1-6.41V4c0-.55-.45-1-1-1s-1 .45-1 1v4c0 .28.11.53.29.71l2 2a1.003 1.003 0 0 0 1.42-1.42L9 7.59z" fillRule="evenodd" />
                </svg>
              </Blueprint2.Icon>
    -         Due: 18/5, 13:24
    +         Due: 18/5, 5:24
            </div>
          </Text>
          <NavLink to="/academy/sidequests/2/0" activeClassName="active" ariaCurrent="true">
            <Route path="/academy/sidequests/2/0" exact={[undefined]} strict={[undefined]} location={[undefined]}>
              <Link to="/academy/sidequests/2/0" className={[undefined]} style={[undefined]} aria-current={false} replace={false}>

      58 |   )
      59 |   const tree = mount(app)
    > 60 |   expect(tree.debug()).toMatchSnapshot()
      61 | })
      62 |

      at Object.<anonymous>.test (src/components/assessment/__tests__/index.tsx:60:24)

 â€º 1 snapshot test failed.
```

Might be related to #270.

**Comment by geshuming** [comment]
Fixed in #575, timezone issue
